docs/source/deprecation.rst

```diff
@@ -18,7 +18,7 @@
 Deprecation Notice
 ==================
 
-This page provides information about the deprecations of a specific oneDAL functionality.
+This page provides information about the deprecations of a specific Intel(R) oneAPI Data Analytics Library (oneDAL) functionality.
```

Comment:
With move to UXL this should be non Intel oneDAL name as this is generic documentation vs Intel specific. Might be worth broader fix

Reply:
Hi @napetrov should I change this to "oneAPI Data Analytics Library (oneDAL)" instead?